argo_prod/schema.xml

```diff
@@ -8,37 +8,61 @@
 
   <fields>
     <field name="id" type="string" stored="true" indexed="true" multiValued="false" required="true"/>
+    <!-- DEPRECATED: "long" is a Trie based field type; use solr.LongPointField class instead ... -->
```

Comment:
nearly all of these changes are about 
- Trie fields being deprecated in Solr 8
- unused fields